### DISCUSSION TITLE
Support mermaid

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#52] [Feature Requests]
Thanks for your work. 
This theme is charming. 
I want to use it as  the documentation of my open-source projects. But the only that I need is the support of rendering mermaid.

**Comment by colinwilson** [comment]
Thanks for taking an interest. 🙏

Yes. Mermaid support (via front matter) will be support by the end of this week. 👍

**Comment by SWHL** [comment]
I have deploy lotusdocs in my project, which does not had mermaid. The project link is https://rapidai.github.io/LabelConvert/docs/
Looking forward the support of rendering mermaid.

**Reply by colinwilson** [reply]
The theme now supports Mermaid. 👍

[https://lotusdocs.dev/docs/guides/features/mermaid/](https://lotusdocs.dev/docs/guides/features/mermaid/)

Let me know if you experience any issues?
